client/src/frontend/components/journal/IssueList/index.js

```diff
@@ -3,64 +3,45 @@ import PropTypes from "prop-types";
 import EntityGroup from "global/components/composed/EntityGroup";
 import Utility from "global/components/utility";
 import lh from "helpers/linkHandler";
-import { fixtures } from "helpers/storybook/exports";
 import CountTemplate from "./CountTemplate";
 import * as Styled from "./styles";
 
-// TODO: update once API is in place
-function getCategories(journal) {
-  const issues = fixtures
-    .collectionFactory("issue", 4)
-    .map(issue => issue.data);
-  return [
-    {
-      id: 1,
-      title: "Volume 1",
-      slug: "volume-0",
-      issues
-    },
-    {
-      id: 2,
-      title: "Volume 2",
-      slug: "volume-0",
-      issues
-    }
-  ];
-}
-
 function JournalIssueList({ journal }) {
-  const categories = getCategories(journal) || [];
-  const uncategorizedIssues = [];
+  if (
+    !journal?.relationships.recentJournalVolumes.length &&
+    !journal?.relationships.recentJournalIssues.length
+  )
+    return null;
 
-  if (!categories.length && !uncategorizedIssues.length) return null;
+  const noVolumeIssues = journal.relationships.recentJournalIssues.filter(
+    issue => !issue.attributes.journalVolumeNumber
+  );
 
   return (
     <Styled.Wrapper>
       <div className="container flush">
         <Utility.EntityCount
-          count={8}
+          count={journal.attributes.journalIssuesCount}
           unit="issue"
           customTemplate={(count, unit) => (
             <CountTemplate
               count={count}
               unit={unit}
-              categoryCount={categories.length}
+              categoryCount={journal.attributes.journalVolumesCount}
```

Comment:
For discussion: what's the logic here if there are issues in volumes and not. Are there two count labels, one for each section? Or an option to say something like "There are X issues _and_ Y volumes"